src/components/navBar/navBar.js

```diff
@@ -469,6 +470,10 @@ function MdNavItem($mdAria, $$rAF) {
           scope.$apply();
         });
 
+          attrs.$observe('disabled', function (value) {
+              mdNavItem.disabled = value.length === 0 ? true : !!value;
```

Comment:
This conversion should use the `parseAttributeBoolean()` function from `$mdUtil` to be consistent with other directives like [this one](https://github.com/angular/material/blob/9f198c93af2e883d25bbce9fadf2a3d4ac5f4283/src/components/autocomplete/js/autocompleteController.js#L211).